tests/roles/cinder_adoption/tasks/main.yaml

```diff
@@ -91,25 +101,22 @@
   ansible.builtin.pause:
      seconds: 90
 
-- name: check that Cinder is reachable and its endpoints are defined
+- name: wait for Cinder scheduler to be up and ready
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    alias openstack="oc exec -t openstackclient -- openstack"
-
-    ${BASH_ALIASES[openstack]} endpoint list | grep cinder
-    ${BASH_ALIASES[openstack]} volume type list
-  register: cinder_responding_result
-  until: cinder_responding_result is success
-  retries: 60
+    oc exec -t openstackclient -- openstack volume service list --service cinder-scheduler -c State -f value | grep 'up'
+  register: cinder_scheduler_running_result
+  until: cinder_scheduler_running_result is success
+  retries: 5
```

Comment:
I assume retries / delay values are sufficient according to your tests.
In case we face issues we can think about moving  `5` and `2` as defaults, so we can always tune them without requiring any code change (as they might just be overrides passed to the jobs).

Reply:
The values (retries:5, delay:2) are taken from the cinder-volume and cinder-backup service state checks.
Though these work good for this specific check, other checks like waiting for the pod to be running or endpoint being reachable differs.
But I agree that configuring retries and delays for similar type of tasks would be a useful thing.